src/modules/integrator-guide/pages/accessing-corteza/ql.adoc

```diff
@@ -157,10 +171,7 @@ Returns year from the given timestamp.
 |
 The function returns the date formatted by the provided format.
 
-[NOTE]
-====
-*DevNote* add a format reference.
-====
+Format example: `YYYY-MM-DD HH:MI:SS`
```

Comment:
An example is not a reference.
Luckily, we already have one prepared [here](https://docs.cortezaproject.org/corteza-docs/2022.9/integrator-guide/expr/datetime-formatting.html) but I'm not sure if the syntax is the same between these endpoints and expressions; can you check & confirm

Reply:
I tried this query `DATE_FORMAT(createdAt, '%Y')` in RL prefilter records section and got the error `Could not load record list: pq: argument of AND must be type boolean, not type text`. Got the query from `Record created this month` in chart filters. I also tried YEAR(), QUARTER() and DATE() as well as other date and time formators from the link you pasted - nothing works. Either this syntax isn't supported or I'm not doing something right.

Reply:
My bet based on the error message would be that you're not defining an expression that evaluates to a boolean, rather to a string.
The queries passed into record list filtering need to return a boolean

Reply:
If it's a bug let's get an issue for it but don't let it block this reference

Reply:
Yeah, the problem was in the syntax of the expression. Looked at a couple of examples, re-tested and can confirm that everything is working as it should.